src/plugins/exec/sim-host.js

```diff
@@ -51,36 +54,13 @@ function cordovaItemFromSim(sim) {
     return cordovaItem;
 }
 
-function createEmptyItem() {
-    var labeledValue = new CordovaLabeledValue();
-    labeledValue.label = 'No values saved';
-    labeledValue.value = '';
-    var cordovaItem = new CordovaItem();
-    cordovaItem.appendChild(labeledValue);
-    return cordovaItem;
-}
-
-var hasEmptyItem = false;
-function addEmptyItem() {
-    if (hasEmptyItem) {
-        return;
-    }
-
-    var execList = document.getElementById('exec-list');
-    var sims = savedSims.sims;
-    if (sims.length === 0) {
-        execList.addItem(createEmptyItem());
-        hasEmptyItem = true;
-    }
+function showEmptyLabel() {
+    document.getElementById('empty-label').classList.remove('cordova-hidden');
+    document.getElementById('exec-list').classList.add('cordova-hidden');
 }
 
-function removeEmptyItem() {
-    if (!hasEmptyItem) {
-        return;
-    }
-
-    var execList = document.getElementById('exec-list');
-    execList.removeItem(0);
-    hasEmptyItem = false;
+function hideEmptyLabel() {
+    document.getElementById('empty-label').classList.add('cordova-hidden');
+    document.getElementById('exec-list').classList.remove('cordova-hidden');
```

Comment:
Maybe it is better to save the `empty-label` and `exec-list` elements into variables to re-use, instead of querying the DOM always. We don't need to access these elements properties just one time for initialization only, but also given certain conditions.